irr/include/CMeshBuffer.h

```diff
@@ -137,7 +137,7 @@ class CMeshBuffer final : public IMeshBuffer
 	//! Index buffer
 	SIndexBuffer *Indices;
 	//! Bounding box of this meshbuffer.
-	core::aabbox3d<f32> BoundingBox;
+	core::aabbox3d<f32> BoundingBox{{0, 0, 0}};
```

Comment:
FWIW: personally I would apply this change to the other mesh-related uses too.
(that is `IMeshManipulator`, `SAnimatedMesh`, `SMesh`, `SSkinMeshBuffer`, `SkinnedMesh`)